src/main/kotlin/com/terraformation/backend/i18n/Locales.kt

```diff
@@ -9,10 +9,12 @@ import org.springframework.context.i18n.LocaleContextHolder
 
 object Locales {
   val GIBBERISH = Locale.forLanguageTag("gx")
+  val SPANISH = Locale.forLanguageTag("es")
```

Comment:
For some unknown reason, the Java `Locale` class has constants for English, French, German, Italian, and others, but not for Spanish.